### PR TITLE
Resolve #19327

### LINKED ISSUE #19327: OCI PHP 8.1: trim(): Passing null to parameter #1 ($string) of type string is deprecated
### What steps will reproduce the problem?

Using Yii2 oci with PHP 8.1

### What is the expected result?

No error.

### What do you get instead?

`trim(): Passing null to parameter #1 ($string) of type string is deprecated`

### PR DESCRIPTION
**Pull request by longthanhtran**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | #19327

### CODE REVIEW

File: framework/db/oci/Schema.php [hunk 1]
```diff
@@ -426,7 +426,7 @@ protected function createColumn($column)
                         ) {
                             $defaultValue = substr($defaultValue, 1, -1);
                         } else {
-                            $defaultValue = trim($defaultValue);
+                            $defaultValue = trim((string) $defaultValue);
```
Comment (bizley) [inline framework/db/oci/Schema.php:429]: Wouldn't be better to cast string in line 417? strlen and strncmp and substr all require string to be used, not only trim.

Comment (longthanhtran) [reply framework/db/oci/Schema.php:429]: Thanks, that should be much better than my choice. I will update this shortly.

**Review by samdark — COMMENTED** [review]
Please add a line for changelog.

**Comment by bizley** [comment]
Thanks

**Pull request merged**